tests/Application/Commands/RetrospectivelyMatchTest.php

```diff
@@ -58,7 +61,7 @@ public function testNonNumericDaysBackRefusesToRun(): void
     public function testNonWholeDaysBackIsRounded(): void
     {
         $commandTester = new CommandTester($this->command);
-        $commandTester->execute(['days-back' => '7.5']);
+        $commandTester->execute(['days-back' => 7.5]);
```

Comment:
Inputs from the CLI are always strings unless cast – did this not work with the string value?

Reply:
@NoelLH yeah, running tests gives me this error:

```
TypeError: round(): Argument #1 ($num) must be of type int|float, string given
```

Reply:
@tbg-jasonfung doesn't this suggest the code should be casting to avoid the problem, rather than the test no longer testing for this case which I think can happen? Have you tried running the command for real?